src/frontend/CodeLensProvider.ts

```diff
@@ -17,16 +17,27 @@ class SymbolCodeLens extends CodeLens {
 };
 
 
-export class AntlrCodeLensProvider implements CodeLensProvider {
+export class AntlrCodeLensProvider implements CodeLensProvider {    
+    // from https://github.com/eamodio/vscode-gitlens/blob/806a9f312be3f034ba052a573ed400709a9b6cb3/src/codelens/gitCodeLensProvider.ts#L81-L100
+    private _onDidChangeCodeLenses = new EventEmitter<void>();
+    public get onDidChangeCodeLenses(): Event<void> {
+        return this._onDidChangeCodeLenses.event;
+    }
+    
     constructor(private backend: AntlrFacade) { }
 
+    public reset(reason?: 'idle' | 'saved') {
```

Comment:
This function doesn't reset anything, so a more appropriate name would be: `refresh` or `update`.